app/views/hosts/show.html.erb

```diff
@@ -75,10 +75,15 @@
     <div class="stats-well">
       <h4 class="ca"><%= _("Runtime") %></h4>
       <h6 class="ca"><%= n_("last %s day", "last %s days", @range) % @range %></h6>
-      <div class="chart" data-ajax-url='<%= runtime_host_path(@host, :range => @range) %>' data-on-complete='updateChart'>
-        <%= spinner(_('Loading runtime information ...')) %>
-      </div>
+      <div id="runtime_react"></div>
+      <%= mount_react_component('TimeseriesChart', '#runtime_react', {
```

Comment:
doesn't work with PG:
i'm getting this  : 
![faild_loading_data](https://user-images.githubusercontent.com/11807069/40355171-6336e66a-5dbe-11e8-83c1-5c7c3a30fca5.png)
 Plus the error string is not extracted.

with this error:
```
ActiveRecord::StatementInvalid (PG::UndefinedFunction: ERROR:  operator does not exist: timestamp without time zone > integer
 | LINE 1: ...t') AND "reports"."host_id" = $1 AND (reported_at > 7) ORDER...
 |                                                              ^
 | HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
 | : SELECT "reports".* FROM "reports" WHERE "reports"."type" IN ('ConfigReport') AND "reports"."host_id" = $1 AND (reported_at > 7) ORDER BY "reports"."reported_at" ASC):
```